csrc/velox/column.cpp

```diff
@@ -445,8 +458,27 @@ std::unique_ptr<OperatorHandle> OperatorHandle::fromUDF(
 
   velox::TypePtr outputType =
       velox::resolveFunction(udfName, inputRowType->children());
+
   if (outputType == nullptr) {
-    throw std::runtime_error("Request for unknown Velox UDF: " + udfName);
+    auto children = inputRowType->children();
+
+    std::string signature;
+    folly::join(", ", children.begin(), children.end(), signature);
```

Comment:
nit: we may want to get rid of folly as deps in a long term (creates trouble for OSS packaging, etc). Perhaps also use `std::stringstream` similar to `udfSignaturesToString` ?

Reply:
sure lol I did that in the initial version then thought that folly one-liners were more neat. How about using boost though? Do we prefer not using that too?

Reply:
Boost sounds better than folly 😃